webapp/channels/src/components/announcement_bar/announcement_bar_controller.tsx

```diff
@@ -102,6 +103,10 @@ class AnnouncementBarController extends React.PureComponent<Props> {
             <>
                 {adminConfiguredAnnouncementBar}
                 {errorBar}
+                <PostLimitsAnnouncementBar
+                    license={this.props.license}
+                    userIsAdmin={this.props.userIsAdmin}
+                />
```

Comment:
Don't remember the details on this order but will this show the user limit banner before the post one if both limits are surpassed?

Reply:
The furthur down the component, the higher priority it takes. Its always confusing so I've now added documentation there.